final_prep/annotations.rb

```diff
@@ -3,41 +3,72 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# method initiated called build_a_bear including 5 parameters (name, age, fur, clothes, special_power)
 def build_a_bear(name, age, fur, clothes, special_power)
+#Declare a local variable. Assign local variable to string with name paramteter
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #Declare local variable "demographics" with array containing two parameters "name and age"
   demographics = [name, age]
+  #Local variable declared "power_saying" with string value incorporating/interpolating special_power inside
   power_saying = "Did you know that I can #{special_power}?"
+  # hash created for built_bear
   built_bear = {
+    #assign "basic info" to key value of "demographics"
     'basic_info' => demographics,
+    #assign value "clothes" to key "clothes"
     'clothes' => clothes,
+    #assign key "exterior" to value "fur"
     'exterior' => fur,
+    #assign key "cost" to value integer "49.99"
     'cost' => 49.99,
+    #assign array with local variables "greeting" and "power_saying", also the string "Goodnight my friend!"
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
+    #assign boolean value of true to "is_cuddly "
     'is_cuddly' => true,
   }
+  # finalize with built_bear
   return built_bear
 end
-
+# pass arguments "Fluffy", 4, "brown", ["pants", "jorts", "tanktop"], and "give you nightmares"
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+# pass arguments "Sleepy", 2, "purple", ["pajamas", "sleeping cap"], "sleeping in"
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
-
+#Method created "fizzbuzz" and parameters (num_1, num_2, range)
 def fizzbuzz(num_1, num_2, range)
+  # Invokes the for-loop method "each" with rnage of interger
   (1..range).each do |i|
+    # varriable "i" against parameter "num_1" is the
+    # same class as the interger 0
+    # varriable "i" into the parameter "num_2" is the
+    # same class as the interger 0
+    # The logical AND operator checks if both === methods returned TRUE
```

Comment:
I would call these conditions, not methods. I also suggest doing some research on the `%` operator and discussing with classmates- it is a tricky one, but a great opportunity to practice technical explanation.